.travis.yml

```diff
@@ -99,6 +99,10 @@ matrix:
         - "sudo service postgresql restart 10"
     - rvm: 2.5.3
       env: "GEM=actionview:ujs"
+    - rvm: 2.6.0
+      env: "GEM=actionview:ujs"
+    - rvm: ruby-head
+      env: "GEM=actionview:ujs"
```

Comment:
The UJS tests don’t cover any Ruby code, so they don’t need to run against all supported Ruby versions.